Source/Scene/Cesium3DTileset.js

```diff
@@ -892,13 +879,8 @@ define([
 
     function raiseLoadProgressEvents(tileset, frameState) {
         var eventsToRaise = tileset._loadProgressEventsToRaise;
-        var length = eventsToRaise.length;
-        for (var i = 0; i < length; ++i) {
-            var numberOfPendingRequests = eventsToRaise[i].numberOfPendingRequests;
-            var numberProcessing = eventsToRaise[i].numberProcessing;
-
-            frameState.afterRender.push(evenMoreComplicated(tileset, numberOfPendingRequests, numberProcessing));
-        }
+        frameState.afterRender.push(evenMoreComplicated(tileset, tileset._statistics.numberOfPendingRequests,
```

Comment:
You should still pass a function to `frameState.afterRender`, it will be a function with no arguments like the one `evenMoreComplicated` returns.

```javascript
`frameState.afterRender.push(function(){...});
```